deploy/test-environments/fleet_api/src/api/agent_policy_api.py

```diff
@@ -37,7 +37,7 @@ def create_agent_policy(cfg: Munch, json_policy: dict) -> str:
         logger.error(
             f"API call failed, status code {api_ex.status_code}. Response: {api_ex.response_text}",
         )
-        return ""
+        raise api_ex
```

Comment:
I think it's a good that you changed it to follow the function signature and docstring (`Raises:
        APICallException: If the API call fails or returns a non-200 status code.`) but who will handle it? we need to wrap this function calls with a try-catch if we don't want to expect